bft/bft_interp/src/lib.rs

```diff
@@ -223,6 +224,51 @@ where
     pub fn decrement_cell(&mut self) {
         self.cells[self.head].decrement()
     }
+
+    /// Read a value from `file` into memory at the memory pointer
+    ///
+    /// If an I/O Error occurs while trying to read the file, it returns that error wrapped inside a VMError.
+    ///
+    /// Because I couldn't think of what brainfuck should do if it ever runs out of bytes reading stdin, it also
+    /// errors if zero bytes are read.
```

Comment:
This is implementation defined, so it's absolutely fine for you to do this.  It's *excellent* however that you documented it, well done.